client_cli/Cargo.toml

```
@@ -22,6 +22,11 @@ is-it-maintained-issue-resolution = { repository = "https://github.com/hyperledg
 is-it-maintained-open-issues = { repository = "https://github.com/hyperledger/iroha" }
 maintenance = { status = "actively-developed" }
 
+[features]
+vendored = [
+    "dep:openssl-sys"
```

```suggestion
    "dep:iroha_crypto/openssl-sys"
```

I think you should do it like this and remove explicit dependency for `openssl-sys` from this file

---

but why would we need this at all if `openssl` is only used for `dev-dependencies`?